### PR TITLE
Rule Dev

### PR DESCRIPTION
**Pull request by nasbench**
This PR covers the following:

- Add new rules related to [Follina Exploit Leads to Domain Compromise](https://thedfirreport.com/2022/10/31/follina-exploit-leads-to-domain-compromise/):
  - NetSupport Manager Service Install
  - Remote Utilities Host Service Install
  - Suspicious Scheduled Task Name As GUID
  - Suspicious Regsvr32 Execution From Remote Share
- Added more browser processes to the rules that use them (see `Files Changed` section)
- Added more LOLBINs to [f3d39c45-de1a-4486-a687-ab126124f744](https://github.com/SigmaHQ/sigma/blob/1f8e37351e7c5d89ce7808391edaef34bd8db6c0/rules/windows/process_creation/proc_creation_win_sdiagnhost_susp_child.yml)
- Added missing flag to [6a69f62d-ce75-4b57-8dce-6351eb55b362](https://github.com/SigmaHQ/sigma/blob/1f8e37351e7c5d89ce7808391edaef34bd8db6c0/rules/windows/process_creation/proc_creation_win_esentutl_webcache.yml) - (This should've been fixed with the `win-dash` operator)
- Added more Image extensions to [089fc3d2-71e8-4763-a8a5-c97fbb0a403e](https://github.com/SigmaHQ/sigma/blob/1f8e37351e7c5d89ce7808391edaef34bd8db6c0/rules/windows/process_creation/proc_creation_win_susp_regsvr32_image.yml)
- Some generic updates to optimize the selection of some rules

### CODE REVIEW

File: rules/windows/process_creation/proc_creation_win_susp_guid_task_name.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    - attack.t1053.005
+logsource:
+    product: windows
+    category: process_creation
+detection:
+    selection_img:
+        Image|endswith: '\schtasks.exe'
+        CommandLine|contains|all:
+            - '/Create '
+            - '}"' # Ending of the name to avoid possible FP in the rest of the commandline
```
Comment (frack113) [inline rules/windows/process_creation/proc_creation_win_susp_guid_task_name.yml:21]: selection_tn have simple and double quote but the end is double quote only.
So `/Create /TN '{guid}'` may not be cauth

Do you think windows CommandLine is quite permissive for a mix like  `/Create /TN "{guid}'` ?

Comment (nasbench) [reply rules/windows/process_creation/proc_creation_win_susp_guid_task_name.yml]: You're right. It should have the other cases. I have pushed an update.

File: rules/windows/process_creation/proc_creation_win_susp_regsvr32_image.yml [hunk 2]
```diff
@@ -1,23 +1,42 @@
 title: Suspicious Regsvr32 Execution With Image Extension
 id: 089fc3d2-71e8-4763-a8a5-c97fbb0a403e
 status: experimental
-description: utilizes REGSVR32.exe to execute this DLL masquerading as a Image file
+description: Detects execution of REGSVR32.exe with DLL masquerading as Image files
```
Comment (phantinuss) [inline rules/windows/process_creation/proc_creation_win_susp_regsvr32_image.yml:4]:
```suggestion
description: Detects execution of REGSVR32.exe with DLL masquerading as image files
```

File: rules/windows/process_creation/proc_creation_win_susp_regsvr32_remote_share.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+detection:
+    selection_img:
+        - Image|endswith: '\regsvr32.exe'
+        - OriginalFileName: '\REGSVR32.EXE'
+    selection_cli:
+        CommandLine|contains: ' \\\\'
+    condition: all of selection_*
+falsepositives:
+    - Unknown
+# Decrease to medium if this is something common in your org
```
Comment (phantinuss) [inline rules/windows/process_creation/proc_creation_win_susp_regsvr32_remote_share.yml:24]: Or disable the rule altogether. I would say this is true for any rule, thus not a necessary comment.

Comment (nasbench) [reply rules/windows/process_creation/proc_creation_win_susp_regsvr32_remote_share.yml:24]: True. My mindset here was for some rules the line between high and medium is thin depending on the org. I just added it for people who perhaps use this (even though I don't think it's common)

Comment (phantinuss) [reply rules/windows/process_creation/proc_creation_win_susp_regsvr32_remote_share.yml:24]: I don't mind it being there. It was just a general remark.

**Review by phantinuss — APPROVED** [review]

**Pull request merged**